assets/quick-add-bulk.js

```diff
@@ -52,6 +57,18 @@ class QuickAddBulk extends HTMLElement {
     });
   }
 
+  resetQuantityInput(id) {
```

Comment:
Should we be resetting these ourselves? This should come back from the BE form what I understand

Reply:
It seems like this was already done in cart for a while. Ill try to gain more context around this